space_age.py

```diff
@@ -0,0 +1,33 @@
+class SpaceAge(object):
+    __init__(self, seconds):
+        self.seconds = seconds
+        self.planets = {
+            'Earth': 1, 'Mercury': 0.2408467, 'Venus': 0.61519726,
+            'Mars': 1.8808158, 'Jupiter': 11.862615, 'Saturn': 29.447498,
+            'Uranus': 84.016846, 'Neptune': 164.79132
+        }
+
+    def on_mercury(self):
+        return self.seconds/60/60/24/365*self.planets.get('Mercury')
```

Comment:
@zarniak `planets['Mercury']` is a shortcat for dictionary value access.

Reply:
@zarniak Here we can clean up some repetition! How about making a constant, let's say `SECONDS_IN_EARTH_YEAR = 60 * 60 * 24 * 365` and re-using in each method?